### PR TITLE
fix: update brave wallet identity check

### PR DESCRIPTION
**Pull request by josheleonard**
### Description
Updates the identity check for Brave Wallet.

Screen Shot 2022-04-05 at 4 04 28 PM

### Checklist
- [X] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [X] The box that allows repo maintainers to update this PR is checked
- [X] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

**Comment by aaronbarnardsound** [comment]
Hey @josheleonard, thank you for this PR.

Do you happen to know if this change is backwards compatible with older versions of the Brave browser. My understanding is that the `isBraveWallet` flag was not previously attached to the Brave provider, so was wondering if this would break users on older versions?

**Comment by josheleonard** [comment]
@aaronbarnardsound, with this change, versions of Brave without the isBraveWallet variable will fallback to showing connect with MetaMask.

File: packages/demo/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@web3-onboard/core": "^2.0.4",
     "@web3-onboard/fortmatic": "^2.0.0",
     "@web3-onboard/gnosis": "^2.0.0",
-    "@web3-onboard/injected-wallets": "^2.0.0",
+    "@web3-onboard/injected-wallets": "../injected",
```
Comment (aaronbarnardsound) [inline packages/demo/package.json:28]:
```suggestion
    "@web3-onboard/injected-wallets": "^2.0.3",
```

File: packages/injected/src/icons/brave.ts [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+<svg width="218px" height="256px" viewBox="0 0 218 256" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (aaronbarnardsound) [inline packages/injected/src/icons/brave.ts:2]:
```suggestion
<svg height="100%" viewBox="0 0 218 256" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
For all of our icons the largest dimension needs to be set to 100%, so in this case the height is larger, so we set that to 100% and then the width will scale automatically. This allows us to scale the icons for all of the UI.

**Review by aaronbarnardsound — CHANGES_REQUESTED** [review]
Just a couple of small changes and then I think we will be good to go.

**Review by aaronbarnardsound — APPROVED** [review]
Looks good

**Comment by aaronbarnardsound** [comment]
@josheleonard In what cases is the `isBrave` flag added to the provider? I am on the latest version and the provider does not have that flag.

**Comment by josheleonard** [comment]
@aaronbarnardsound, currently the `isBraveWallet` provider flag is only present in Beta builds. This flag will be present in release builds soon.

**Pull request merged**